### PR TITLE
Add tooltip for objects in object browser

### PR DESCRIPTION
**Pull request by chrjorgensen**
### Changes

This PR is a continuation of the tooltips changes - it adds a tooltip to objects in the object browser, when the object is not a source file:

image

### Checklist

* [x] have tested my change

### CODE REVIEW

File: src/locale/ids/fr.ts [hunk 1]
```diff
@@ -30,6 +30,9 @@ export const fr: Locale = {
   'size': 'Taille',
   'modified': 'Modifié',
   'owner': 'Propriétaire',
+  'type': 'Type',
```
Comment (chrjorgensen) [inline src/locale/ids/fr.ts:33]: @sebjulliand We need your translation skills here! 😍

File: src/locale/ids/fr.ts [hunk 2]
```diff
@@ -30,6 +30,9 @@ export const fr: Locale = {
...
+  'attribute': 'Attribute',
+  'created_by': 'Created by',
```
Comment (chrjorgensen) [inline src/locale/ids/fr.ts:35]: @sebjulliand We need your translation skills here! 😉

Comment (sebjulliand) [inline src/locale/ids/fr.ts:35]: Voilà!
```suggestion
  'type': 'Type',
  'attribute': 'Attribut',
  'created_by': 'Créé par',
```

**Review by sebjulliand — APPROVED** [review]
Tested over a few different objects and it works like a charm.
A fine addition! I guess we have big enough tooltips now 😅

I'll approve and let you merge whenever you want, if you'd like others to review too. But I'm fine with it!

**Comment by chrjorgensen** [comment]
@sebjulliand

> I guess we have big enough tooltips now 😅

I remember someone recently telling me "we can't have enough information"... 😉 

Just kidding - I agree and had to think about when deciding which information to include. We really need that FS extension...! 😍

**Pull request merged**